schedule/rt/online-migration-slertnext.yaml

```diff
@@ -2,7 +2,6 @@
 description: 'Maintainer: thehejik; upgrade from SLERT12 SP3 to SLERT 12 SP4'
 name: 'online-migration-slert15spnext@64bit ( qemu )'
 schedule:
-  - installation/isosize
```

Comment:
"rt: Remove isosize from installation schedule": out of curiosity why? (might be worth to put the reason to the commit message).

Reply:
Commit msg is now: We don't need to do isosize check for Online/Full mediums. It was mainly used to check that
iso will fit to CD/DVD, which is not relevent now.  Is it OK?